joomla/joomla-cms
# [5.4] CMSPlugin: deprecation for registerListeners

Pull Request for Issue # .

### Summary of Changes

The PR deprecate `CMSPlugin::registerListeners()` as no longer needed when plugin will implement `SubscriberInterface` 
(I wonder why it was not marked as deprecated before).

### Testing Instructions

Apply PR, and navigate around the site, all should work as before.

The PR requires review from maintainers.

### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/422

Ping @HLeithner @wilsonge
